benchs/Fake/Text.hs

```diff
@@ -8,6 +8,7 @@ module Fake.Text
     , filter
     , reverse
     , decimal
+    , double
```

Comment:
It's just a contraption to allow building without external dependencies like text, bytestring, etc.. when doing internal benchs (instead of comparative benchs with other libraries).
The fake module limits the number of ifdef in the bench modules.